BUILD.bazel

```diff
@@ -37,3 +39,4 @@ filegroup(
     ]),
     visibility = ["//visibility:public"],
 )
+
```

Comment:
can you do a pass through your PRs in general to remove spurious unneeded deltas so the review is easier?